modules/services/listenbrainz-mpd.nix

```
@@ -40,6 +40,10 @@ in {
         ExecStart = "${cfg.package}/bin/listenbrainz-mpd";
         Restart = "always";
         RestartSec = 5;
+        Type = if (lib.versionAtLeast cfg.package.version "2.3.2") then
```

Can remove the unnecessary parentheses
```suggestion
        Type = if lib.versionAtLeast cfg.package.version "2.3.2" then
```